`distributed/tests/test_worker_memory.py::test_digests` flaky

e.g. https://github.com/dask/distributed/actions/runs/3820613535/jobs/6499032908


```
FAILED distributed/tests/test_worker_memory.py::test_digests - AssertionError: assert 'disk-write-spill-duration' not in defaultdict(<class 'float'>, {'latency': 0.016292095184326172, 'disk-write-spill-duration': 0.0050070359998244385, 'compute-duration': 3.504753112792969e-05, 'get-data-send-duration': 0.0032188892364501953})
 +  where defaultdict(<class 'float'>, {'latency': 0.016292095184326172, 'disk-write-spill-duration': 0.0050070359998244385, 'compute-duration': 3.504753112792969e-05, 'get-data-send-duration': 0.0032188892364501953}) = <Worker 'tcp://127.0.0.1:65106', name: 0, status: running, stored: 2, running: 0/1, ready: 0, comm: 0, waiting: 0>.digests_total
```

@crusaderky I believe you added this recently. Can you have a look?

--- a/distributed/tests/test_worker_memory.py
+++ b/distributed/tests/test_worker_memory.py
@@ -1062,8 +1062,10 @@ async def test_release_evloop_while_spilling(c, s, a):
     worker_kwargs={"memory_limit": "100 MiB"},
     # ^ must be smaller than system memory limit, otherwise that will take precedence
     config={
-        "distributed.worker.memory.target": 0.5,
-        "distributed.worker.memory.spill": 1.0,
+        "distributed.worker.memory.target": 0.5,  # 50 MiB
+        # 97 GiB. It must be extremely high otherwise there's a risk we'll spend time
+        # trying to gc before get_process_memory gets patched below
+        "distributed.worker.memory.spill": 1000,
         "distributed.worker.memory.pause": False,
         "distributed.worker.memory.monitor-interval": "10ms",
     },
@@ -1072,7 +1074,7 @@ async def test_digests(c, s, a, b):
     trigger_spill = False
 
     def get_process_memory():
-        return 2**30 if trigger_spill else 0
+        return 1001 * 100 * 2**20 if trigger_spill else 0
 
     a.monitor.get_process_memory = get_process_memory
 

